src/net.h

```diff
@@ -856,6 +856,14 @@ class CConnman
     uint64_t GetTotalBytesRecv() const;
     uint64_t GetTotalBytesSent() const EXCLUSIVE_LOCKS_REQUIRED(!m_total_bytes_sent_mutex);
 
+    struct MsgStatsValue {
+        int msg_count;
+        uint64_t byte_count;
```

Comment:
May want to use std::atomic_uint64_t instead.

Need to make a list of pros and cons (atomic_uint64_t vs uint64_t)

Reply:
Resolved by the suggestions in https://github.com/vasild/bitcoin/commits/getnetmsgstats

Have some notes on this as well. The main thing is even though atomic_uint64_t comes for free with 64 bit machines, you need to be explicit if you want this behavior on 32 bit machines as well.